package.json

```diff
@@ -145,6 +145,7 @@
   "lint-staged": {
     "linters": {
       "*.{ts,tsx}": [
+        "yarn format:prettier",
```

Comment:
So [prettier is run as part of ESLint](https://github.com/Workday/canvas-kit/blob/master/eslintrc.js#L82), which is why this was left out. I would expect this to be running prettier twice. However, there do seem to be some formatting changes that have been picked up, so I'm not quite sure why there would be a difference. Any ideas?

Reply:
Probably this: https://github.com/Workday/canvas-kit/blob/master/eslintrc.js#L1 😄 
I'll fork and put up a new PR.